code-studio/src/css/plc/teacher_dashboard.scss

```diff
@@ -64,25 +64,48 @@
       margin: 10px;
       padding: 5px;
 
+      background-color: #0094CA;
+      color: white !important;
+
+      position: relative;
+
+      .ribbon-wrapper {
+        width: 90px;
+        height: 100%;
+        overflow: hidden;
+        position: absolute;
+        top: 0px;
+        right: 0px;
+
+        .ribbon {
+          -webkit-transform: rotate(45deg);
+          -moz-transform:    rotate(45deg);
+          -ms-transform:     rotate(45deg);
+          -o-transform:      rotate(45deg);
```

Comment:
I see vendor-prefixed `transform` but not un-prefixed.  I believe we have a SCSS mixin that will generate the vendor prefixes.